assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceSegmentationIntroductoryOverlayNotification.js

```diff
@@ -83,7 +100,7 @@ export default function AudienceSegmentationIntroductoryOverlayNotification() {
 				<Button
 					tertiary
 					disabled={ isDismissing }
-					onClick={ dismissNotification }
+					onClick={ scrollToWidgetAndDismissNotification }
```

Comment:
The click handler is applied to the wrong button, this is the "Got it" button, it should be applied to the "Show me" button.